dockerfiles/base/scripts/base/commands/cmd_stop.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# Copyright (c) 2012-2016 Codenvy, S.A.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#   Tyler Jewell - Initial Implementation
+#
+
+cmd_stop() {
+  debug $FUNCNAME
+  FORCE_STOP=false
+  if [[ "$@" == *"--skip:graceful"* ]]; then
+  	FORCE_STOP=true
+  fi
+
+  if server_is_booted $(get_server_container_id $CHE_CONTAINER_NAME); then 
+    if [[ ${FORCE_STOP} = "false" ]]; then
+      info "stop" "Stopping workspaces..."
+      if ! $(cmd_lifecycle action "graceful-stop" "$@" >> "${LOGS}" 2>&1 || false); then
```

Comment:
if we drop output of the graceful-stop we won't see the upcoming error message saying that authentication needs to be used. I would expect that it's directly printed on the screen without any user action on having to check cli.log (also ppl may not know directly where the file is located by using only `cli.log`)